src/frontend/src/components/core/parameterRenderComponent/index.tsx

```diff
@@ -296,19 +295,7 @@ export function ParameterRenderComponent({
   };
 
   return useMemo(
```

Comment:
useMemo is no longer needed as it is no longer dependent on the listed dependencies